Apps/W1/HybridGP/app/src/Migration/Support/CheckBooks/GPCheckbookTransactions.table.al

```diff
@@ -216,8 +216,132 @@ table 40101 "GP Checkbook Transactions"
         }
     }
 
-    procedure MoveStagingData(CheckbookId: Code[15]; PostingGroup: Code[20])
+
+    procedure MoveStagingData(BankAccountNo: Code[20]; BankAccPostingGroupCode: Code[20]; CheckbookID: Text[15])
+    var
+        GenJournalLine: Record "Gen. Journal Line";
+        GenJournalBatch: Record "Gen. Journal Batch";
+        DataMigrationFacadeHelper: Codeunit "Data Migration Facade Helper";
+        AccountNo: Code[20];
+    begin
+        SetRange(CHEKBKID, CheckbookID);
+        if FindSet() then
+            repeat
+                if CMTrxType = 2 then begin
```

Comment:
Is the 'begin end' necessary here since it isn't a compound statement?

Reply:
? not seeing which statement you are referring too. 'if FindSet() then'? the 'if CMTrxType =2' is a compound statement. Or do you want the AccountNo value set first and then change it if CMTrxType = 2?